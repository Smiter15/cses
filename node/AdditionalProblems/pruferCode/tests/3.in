10
2 10 6 3 10 2 3 7
