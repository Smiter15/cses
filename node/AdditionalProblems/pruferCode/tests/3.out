2 1
10 4
6 5
3 6
10 8
2 9
3 2
7 3
7 10 
